debian/control

```diff
@@ -0,0 +1,33 @@
+Source: txfixtures
+Maintainer: Debian Python Modules Team <python-modules-team@lists.alioth.debian.org>
+Uploaders: Free Ekanayaka <freee@debian.org>
+Section: python
+Priority: optional
+Build-Depends: dh-python,
+               debhelper (>= 9),
+               python3-all,
+               python3-setuptools,
+               python3-fixtures,
+               python3-testtools,
+               python3-twisted,
+	       python3-systemfixtures (>= 0.6.2),
+	       python3-psutil,
+	       python3-selenium,
+	       procps,
+Standards-Version: 3.9.8
+Homepage: https://launchpad.net/txfixtures
+Vcs-Browser: https://anonscm.debian.org/cgit/python-modules/packages/txfixtures.git
+Vcs-Git: git://anonscm.debian.org/python-modules/packages/txfixtures.git
```

Comment:
The linked repos have not been updated for 7 years.

Reply:
Updating them to point to Launchpad and GitHub accordingly.